src/main/java/org/myrobotlab/framework/Service.java

```diff
@@ -2813,5 +2813,14 @@ public boolean isPeerStarted(String peerKey) {
   protected void registerForInterfaceChange(Class<?> clazz) {
     Runtime.getInstance().registerForInterfaceChange(getClass().getCanonicalName(), clazz);
   }
+  
 
+  /**
+   * default emtpy getDefaultConfigs - will be overriden by services which provide sane defaults
+   * @return
+   */
+ static public Map<String, Map<String, ServiceConfig>> getDefaultConfigs() {
```

Comment:
I thought we agreed were going to drop the outer map ?

Reply:
I still think that this method signature should return a simple  List<ServiceConfig>  and that Service config should contain the "name" of the service as a member.  If we don't want to include name in the service config, then we should return something like  List<Pair<String, ServiceConfig>>   where the pair is a name/config pair.  The order of this list should determine the runtime start order..  and runtime should not be included in this returned list of service configs.